src/index.tsx

```diff
@@ -4,15 +4,15 @@ import Step from 'shepherd.js/src/types/step';
 import Tour from 'shepherd.js/src/types/tour';
 
 interface ShepherdProps {
-  steps: Array<Step>;
+  steps: Array<Step.StepOptions>;
   tourOptions: Tour.TourOptions;
 }
 
 interface ShepherdStepWithType extends Step.StepOptionsButton {
   type?: string;
 }
 
-const ShepherdTourContext = React.createContext({});
+const ShepherdTourContext = React.createContext(Tour);
```

Comment:
I might be confused but I'm not sure this is actually what you want? The `ShepherdTourContext` is supposed to contain an instance of `Tour`, whereas you're defaulting it to class object here instead. I think a more suitable alternative would be

```ts
React.createContext<Tour | null>(null);
```

which represents what is actually happening.

This would require people to check there's actually an active tour when consuming the context, but they should have been doing that anyway if value could have been `{}` instead of an instance of the tour.

Reply:
If you don't want to break people's code who haven't been checking for this, you could do 

```ts
React.createContext<Tour | {}>({});
```

though using `{}` as a default doesn't seem sensible in the long term.

Reply:
Since we're major bumping the Shepherd version next, we'll just make this a breaking change.